### PR TITLE
Update templates.json

### PR DESCRIPTION
**Pull request by ln-dev7**
Added a web application that retrieves information from a GitHub profile

### CODE REVIEW

**Review by ln-dev7 — COMMENTED** [review]
edit npm: ""

File: src/routes/templates/templates.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     		"title": "svelte-app",
    		"url": "https://github.com/ln-dev7/userGithub",
     		"description": "web application that retrieves information from a GitHub profile",
-    		"npm": "",
+    		"npm": "npm run dev",
```
Comment (MacFJA) [inline src/routes/templates/templates.json:6]: The data behind this key is the package name (as available on NPM repository), not the command to run.

For a template, the `npm` key don't have a lot of meaning thought

File: src/routes/templates/templates.json [hunk 2]
```diff
@@ -1,4 +1,16 @@
 [
+	{
+    		"title": "svelte-app",
```
Comment (MacFJA) [inline src/routes/templates/templates.json:3]: `svelte-app` is a bit too generic for a template name.

I believe that (almost) all templates are for building an app, so they all are a "Svelte App".

Can you choose a name that better describe your template ?

**Review by MacFJA — CHANGES_REQUESTED** [review]

**Comment by benmccann** [comment]
closing due to lack of response

**Pull request closed without merging**